src/qt/button.cpp

```diff
@@ -46,7 +46,22 @@ bool wxButton::Create(wxWindow *parent, wxWindowID id,
 {     
     QtCreate(parent);
     SetLabel( wxIsStockID( id ) ? wxGetStockLabel( id ) : label );
-
```

Comment:
There are spaces on this row. You can see that by simply looking at this diff on github, and observe that this row is marked as different from the old one.